TelepresenceBot/core/src/main/java/com/novoda/tpbot/bot/device/usb/UsbChangesListener.java

```diff
@@ -1,6 +1,6 @@
-package com.novoda.tpbot.bot;
+package com.novoda.tpbot.bot.device.usb;
 
-interface MovementService {
+interface UsbChangesListener {
```

Comment:
This was confusing before, it didn't represent the `MovementService`. Now it represents changes in Usb connectivity and is implemented by the `UsbDeviceConnection`.